django/db/migrations/autodetector.py

```diff
@@ -17,6 +18,22 @@
     resolve_relation,
 )
 
+OperationDependency = namedtuple(
+    "OperationDependency", "app_label model_name field_name type"
+)
+
+
+class DependencyType(Enum):
+    CREATE = 0
+    REMOVE = 1
+    ALTER = 2
+    ORDER_WRT_UNSET = 3
+    FOO_TOGETHER_CHANGE = 4
+
+    # A similar format was proposed for Python 3.10.
+    def __repr__(self):
+        return f"{self.__class__.__qualname__}.{self._name_}"
+
```

Comment:
I'm not sure we really need to copy this around? It feels like we're trying to fight Python's default everywhere - in the end, this format that _was_ proposed for Python 3.10 and reverted didn't materialise and something slightly different happened in the form of `ReprEnum` in 3.11. There is something to say about the backward compatibility now in `TextChoices`, etc. but I think we shouldn't continue this elsewhere. And I also don't think we actually care about this here as we're only comparing constant values.

```suggestion
```

Reply:
Agreed, removed.